selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/gwt/CheckSimpleGwtAppTest.java

```diff
@@ -40,25 +50,53 @@
 public class CheckSimpleGwtAppTest {
   private static final String BUILD_COMMAND = "build";
   private static final String RUN_GWT_COMMAND = "runGwt";
+  private static final String GWT_CODESERVER_NAME = "gwt-codeserver";
+  private static final String WS_CONFIG_PATH =
+      CheckSimpleGwtAppTest.class
+          .getResource("/templates/workspace/" + CODENVY_UBUNTU_JDK8)
+          .getPath();
 
   @Inject private Ide ide;
   @Inject private ProjectExplorer projectExplorer;
   @Inject private ToastLoader toastLoader;
   @Inject private Consoles consoles;
 
-  @InjectTestWorkspace(memoryGb = 4)
   private TestWorkspace testWorkspace;
 
   @Inject private TestCommandServiceClient testCommandServiceClient;
   @Inject private TestWorkspaceServiceClient workspaceServiceClient;
   @Inject private TestProjectServiceClient testProjectServiceClient;
-
+  @Inject private TestUser testUser;
   private String projectName;
 
   @BeforeClass
   public void prepare() throws Exception {
     projectName = NameGenerator.generate("project", 4);
 
+    WorkspaceConfigDto workspace =
```

Comment:
Please, reuse utility for deserialization

Reply:
ok